src/modules/storefront/components/Footer.tsx

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable react/jsx-key */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import { ReactComponent as FacebookIcon } from '../../shared/assets/icons/facebook.svg';
+import { ReactComponent as InstagramIcon } from '../../shared/assets/icons/instagramFilled.svg';
+import { ReactComponent as MessageIcon } from '../../shared/assets/icons/message.svg';
+import { ReactComponent as TwitterIcon } from '../../shared/assets/icons/twitter.svg';
+import { Link } from '../../shared/components/Link';
+import { FooterData, FooterDefault } from '../interfaces';
+export const Footer = ({
+  data,
+  defaultData,
+}: {
+  data: FooterData;
+  defaultData: FooterDefault;
+}) => {
+  const backgroundColor = data.bgColor || defaultData.bgColor;
+  const color = data.textColor || defaultData.textColor;
+
+  const socialNetWorkLinkSharedClassnames =
+    'pw-fill-[#090909] hover:pw-fill-brand-primary pw-w-4 pw-h-4 pw-p-2';
+
+  const typeToIcon = new Map([
+    [
+      'facebook',
+      // eslint-disable-next-line react/jsx-key
+      <FacebookIcon
+        className={socialNetWorkLinkSharedClassnames}
+        style={{ fill: color, background: '#090909', borderRadius: '50%' }}
```

Comment:
O ideal seria usar classes o máximo possível, por exemplo o `background: '#090909' poderia ser escrito `pw-bg-[#090909]`, mas quando for alguma variável, como o `fill: color`, pode usar o `style` mesmo